Nos/Views/UNS/SetUpUNSBanner.swift

```diff
@@ -11,65 +11,91 @@ struct SetUpUNSBanner: View {
     
     var action: () -> Void
     
+    private var backgroundGradient: LinearGradient {
```

Comment:
This file has a lot of implicit colors, we should define them in the Assets file instead

Reply:
Ah good catch. So I'm in a bit of a catch-22 here. I updated to macOS Sonoma so I can't run Xcode 14.3.1, and the SwiftGen script isn't working in Xcode 15. I fixed this on my `uns-redesign` branch but that hasn't been merged yet so I made a ticket #582 to do this when I get back from vacation.